------------------------------------------------------------------------
r77542 | vboxsync | 2019-03-03 05:40:17 +0100 (nie, 03 mar 2019) | 1 line

linux/vboxsf: MS_REMOUNT and friends moved to uapi/linux/mount.h in 5.0 (rc8), so we need to include it to use it.  bugref:9172

Index: src/VBox/Additions/linux/sharedfolders/vfsmod.c
===================================================================
--- src/VBox/Additions/linux/sharedfolders/vfsmod.c	(revision 77541)
+++ src/VBox/Additions/linux/sharedfolders/vfsmod.c	(revision 77542)
@@ -45,7 +45,9 @@
 #include "revision-generated.h"
 #include "product-generated.h"
 #include "VBoxGuestR0LibInternal.h"
-#if LINUX_VERSION_CODE < KERNEL_VERSION(3, 3, 0)
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 0, 0)
+# include <uapi/linux/mount.h> /* for MS_REMOUNT */
+#elif LINUX_VERSION_CODE < KERNEL_VERSION(3, 3, 0)
 # include <linux/mount.h>
 #endif
 #include <linux/seq_file.h>

------------------------------------------------------------------------
